src/components/OverviewContent.js

```diff
@@ -1,97 +1,49 @@
-import React, {Component} from "react";
-import {Layout, Divider} from "antd";
+import React, { Component } from "react";
+import { Layout, Divider } from "antd";
 
 import SitePageHeader from "./SitePageHeader";
 import ClinicalSummary from "./overview/ClinicalSummary";
 import VariantsSummary from "./overview/VariantsSummary";
-import OverviewSettingsControl from "./overview/OverviewSettingsControl";
 import { SITE_NAME } from "../constants";
 import ExperimentsSummary from "./overview/ExperimentsSummary";
-import { Modal, Button } from "antd";
 import { DEFAULT_OTHER_THRESHOLD_PERCENTAGE } from "../constants";
 
 class OverviewContent extends Component {
     constructor(props) {
         super(props);
-        const startThreshold =
-          JSON.parse(localStorage.getItem("otherThresholdPercentage")) ?? DEFAULT_OTHER_THRESHOLD_PERCENTAGE;
+        const threshold =
+      JSON.parse(localStorage.getItem("otherThresholdPercentage")) ?? DEFAULT_OTHER_THRESHOLD_PERCENTAGE;
         this.state = {
-            otherThresholdPercentage: startThreshold,
-            previousThresholdPercentage: startThreshold,
-            modalVisible: false,
+            otherThresholdPercentage: threshold,
         };
-        this.setOtherThresholdPercentage = this.setOtherThresholdPercentage.bind(this);
-        this.openModal = this.openModal.bind(this);
-        this.setValueAndCloseModal = this.setValueAndCloseModal.bind(this);
-        this.cancelModal = this.cancelModal.bind(this);
     }
 
     componentDidMount() {
         document.title = `${SITE_NAME} - Overview`;
     }
 
-    setOtherThresholdPercentage(value) {
-        this.setState({otherThresholdPercentage: value});
-    }
-
-    openModal() {
-        this.setState({modalVisible: true});
-        // save threshold in case user cancels
-        this.setState({previousThresholdPercentage: this.state.otherThresholdPercentage});
-    }
-
-    setValueAndCloseModal() {
-        localStorage.setItem("otherThresholdPercentage", JSON.stringify(this.state.otherThresholdPercentage) );
-        this.setState({modalVisible: false});
-    }
-
-    cancelModal() {
-        this.setState({modalVisible: false});
-        this.setState({otherThresholdPercentage: this.state.previousThresholdPercentage});
-    }
-
     render() {
         return (
             <>
```

Comment:
Indentation: this markup wraps the following. It should be at least as indented (and I would say less indented)